packages/twenty-server/src/engine/core-modules/file-storage/drivers/s3.driver.ts

```diff
@@ -417,4 +417,24 @@ export class S3Driver implements StorageDriver {
 
     return true;
   }
+
+  async checkFolderExists(folderPath: string): Promise<boolean> {
+    try {
+      const listCommand = new ListObjectsV2Command({
```

Comment:
does this actually checks that the folder exists or that a folder exists that start with the same prefix?